src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using Arcus.Observability.Telemetry.Core;
+using GuardNet;
+using Microsoft.AspNetCore.Http;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.Extensions.Logging
+{
+    public static class ILoggerExtensions
```

Comment:
I guess it's not necessary to also check the key/values in the context dictionary for null?

Reply:
No because it's optional